### ISSUE TITLE
Vernon Bug Hunt

### ISSUE THREAD

**Issue by vrachal86** [jonathanwalkerscripts/OppChasers#121]
Create an acceptance test-style checklist with the following: 

(Title) Bug Hunt

**On websites/applications of your choosing, see if you can find bugs in the following categories:**

- [ ] Image not appearing correctly
- [ ] Link broken/goes to wrong page
- [ ] Color Contrast not ADA Compliant
- [x] Missing Alt Text 
- [ ] Present Alt Text (Show me a screenshot from Inspect!)
- [ ] Any Bug with Data Fields
- [ ] Design Flaw or Typo
- [ ] Crash or Error Message
- [ ] Google Lighthouse ADA Score of exactly 80
- [ ] Any bug on CEO Works or a company you've worked with

- [x] Bonus: Use the inspect tool to change the headline of a major news story. Post a screenshot in the comments section.

**Comment by vrachal86** [comment]
Missing Alt Text

[Fox Sports](https://www.foxsports.com/stories/nba/russell-westbrook-is-officially-mr-triple-double-but-his-legacy-is-complicated)

Screenshot
![Screenshot (46)](https://user-images.githubusercontent.com/79952835/117912417-596aaf80-b294-11eb-8e4b-c248ea1e47b9.png)

**Comment by vrachal86** [comment]
Bonus

[Fox Sports](https://www.foxsports.com/stories/nba/lebron-james-los-angeles-lakers-injury-return-new-york-knicks-nba-playoffs)

Screenshot
![Screenshot (48)](https://user-images.githubusercontent.com/79952835/117914928-14954780-b299-11eb-8ebf-64d10d1314c4.png)
![Screenshot (49)](https://user-images.githubusercontent.com/79952835/117914932-165f0b00-b299-11eb-88dc-088a14debd23.png)
